app/javascript/controllers/stackmap_controller.js

```diff
@@ -42,8 +42,13 @@ export default class extends Controller {
 
       this.libraryTarget.textContent = map.library
       this.floornameTarget.textContent = map.floorname
-      // Replace <li>- with <li>
-      this.directionsTarget.innerHTML = map.directions.replaceAll(/<li>-\s*/g, '<li>')
+
+      if (map.directions == "") {
+        this.directionsTabTarget.classList.add('d-none')
+      } else {
+        // Replace <li>- with <li>
```

Comment:
```suggestion
        // Replace <li>- with <li>
        this.directionsTabTarget.classList.remove('d-none')
```

I think I've convinced myself we never reuse the rendered partial in any case where we'd have to undo this, right?